composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "dektrium/yii2-user",
+    "name": "dabilite/yii2-user",
     "description": "Flexible user registration and authentication module for Yii2",
```

Comment:
https://snippets.khromov.se/composer-use-your-own-fork-for-a-package/ works - i close this issue.